searx/autocomplete.py

```diff
@@ -36,7 +26,26 @@ def get(*args, **kwargs):
     return http_get(*args, **kwargs)
 
 
-def dbpedia(query, lang):
+def brave(query, _lang):
+    # brave search autocompleter
+    url = 'https://search.brave.com/api/suggest?'
+    url += urlencode({'q': query})
+    country = 'all'
+    # if lang in _brave:
+    #    country = lang
+    kwargs = {'cookies': {'country': country}}
```

Comment:
I see this cookie `ui_lang=en-us` but not `country`.

Reply:
When I go to https://search.brave.com/ and trace the suggestion request, I see:

![grafik](https://user-images.githubusercontent.com/554536/150690928-9726ff62-d60b-4a5f-90e9-a8a97d85333d.png)

Reply:
I see
![image](https://user-images.githubusercontent.com/1594191/150692379-01eeaf35-d81a-43ca-b00d-e3edd1c84d78.png)

may be I use the wrong setting.